modules/standard/Regexp.chpl

```diff
@@ -90,8 +400,31 @@ proc compile(pattern: string, utf8=true, posix=false, literal=false, nocapture=f
   return ret;
 }
 
-/*  Compile a regular expression and retrieve an error code if compilation
-    failed.
+/*  Compile a regular expression. If the optional error argument is provided,
```

Comment:
Something is causing the documentation output to get a little gray box here.  I think it is the extra space, but I'm not sure.  Are you seeing it?